src/components/items-for-hideout/index.js

```diff
@@ -45,12 +48,25 @@ function ItemsForHideout(props) {
     //     </div>
     // }
 
+    const unbuilt = useMemo(() => {
+        return data.filter(module => settings[module.normalizedName] < module.level);
+    }, [data, settings]);
+
     let extraRow = false;
 
     if (data.length <= 0) {
         extraRow = t('No hideout modules requires this item');
+    } else if (unbuilt.length !== data.length && !showAll) {
+        extraRow = (
+            <>
+                {t('No unbuilt hideout moduels for selected filters but some were hidden by ')}<Link to="/settings/">{t('your settings')}</Link>
+            </>
+        );
     }
 
+    let displayList = unbuilt;
+    if (showAll) displayList = data;
```

Comment:
It's a good practice to place the change made by the if on a new line for better readability
```suggestion
    if (showAll)
        displayList = data;
```